webapp/channels/src/actions/admin_actions.jsx

```diff
@@ -38,7 +38,7 @@ export async function adminResetMfa(userId, success, error) {
 }
 
 export async function getClusterStatus(success, error) {
-    const {data, error: err} = await AdminActions.getClusterStatus()(dispatch, getState);
+    const {data, error: err} = await dispatch(AdminActions.getClusterStatus());
```

Comment:
@escofresco thank you for the PR but it doesn't work that way because where would you get the dispatch in the first place, you would have to convert each of getClusterStatus into thunk actions first to be able to use dispatch from its params

Reply:
sorry @M-ZubairAhmed total noob here wrt redux, reviewing https://github.com/mattermost/mattermost-webapp/pull/4402/files#diff-92c78d8d4fa1772d360c0c76d8015539cb890a80eb98390a5e6514cb442b9dd7 from https://github.com/mattermost/mattermost-webapp/pull/11331#issuecomment-1277636398 to get a clearer picture of what the fix is

Reply:
@escofresco feel free to reach out for help